app/views/handlers/relationship_collector.py

```diff
@@ -93,14 +90,16 @@ def handle_post(self):
         )
 
         if self._is_last_relationship():
-            parent_location = Location(
-                section_id=self._current_location.section_id,
-                block_id=self.rendered_block["id"],
-            )
             self.questionnaire_store_updater.add_completed_location(
-                location=parent_location
+                location=self.parent_location
+            )
+            is_path_complete = self.router.is_path_complete(self._get_routing_path())
+
+            self.questionnaire_store_updater.update_section_status(
+                is_complete=is_path_complete,
+                section_id=self.parent_location.section_id,
+                list_item_id=self.parent_location.list_item_id,
             )
```

Comment:
Now that `_get_routing_path` has been overridden, can this go back to calling the original `self._update_section_completeness`?